README.md

```diff
@@ -38,10 +38,18 @@ factors appropriately to yield the 2-to-2 scattering matrix elements.
 
 ### Paclet Installation
 
-**WallGoMatrix** can be installed as a Wolfram Paclet by running the following in Mathematica
+**WallGoMatrix** can be installed as a Wolfram Paclet by running either of the following two commands in Mathematica.
+Depending if the installation happens through the
```

Comment:
What is the reason for these two different ways of installation?

Reply:
One is for the GitHub resource and one for the mathematica repository. Just offering different paths to get it. Same with pip and conda.